README.md

```
@@ -4,31 +4,27 @@
-  | husky v4 `4.3.8`              | `53.5 kB`    | `~1 mB`   |
-  | husky v8 `8.0.3`              | `6.44 kB`    | `6.44 kB` |
-  | pre-commit `1.2.2`            | `~80 kB`     | `~850 kB` |
-  | **simple-git-hooks** `2.11.0` | `10.9 kB`    | `10.9 kB` |
+| Package | Unpacked size | With deps |
+|---------|-------------|-----------|
+| husky `9.1.7` | `4.0 kB` | `4.0 kB` |
+| **simple-git-hooks** `2.13.1` | `13.0 kB` | `13.0 kB` |
+| lefthook `2.1.10` | `28.4 kB` | `28.4 kB` |
+| pre-commit `2.0.0` | `35.9 kB` | `~850 kB` |
 
 ### Who uses simple-git-hooks?
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)